modules/sdk/src/main/scala/org/tessellation/sdk/domain/trust.scala

```diff
@@ -0,0 +1,35 @@
+package org.tessellation.sdk.domain
+
+import cats.syntax.bifunctor._
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.InternalTrustUpdate
+import org.tessellation.security.hex.Hex
+
+import derevo.cats.eqv
+import derevo.derive
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.cats._
+import eu.timepit.refined.numeric.Interval
+import eu.timepit.refined.refineV
+import fs2.data.csv.generic.semiauto.deriveRowDecoder
+import fs2.data.csv.{CellDecoder, DecoderError}
+
+object trust {
+  @derive(eqv)
+  case class TrustRating(peerId: PeerId, rating: TrustRatingRefined) {
+    def toInternalTrustUpdate = InternalTrustUpdate(peerId, rating)
+  }
+
+  object TrustRating {
+    implicit val peerIdDecoder: CellDecoder[PeerId] = CellDecoder.stringDecoder.map(Hex(_)).map(PeerId(_))
+
+    implicit val trustRatingRefinedDecoder: CellDecoder[TrustRatingRefined] =
+      CellDecoder.doubleDecoder.emap(refineV[TrustRatingRefinement](_).leftMap(new DecoderError(_)))
+
+    implicit val rowDecoder = deriveRowDecoder[TrustRating]
+  }
+
+  type TrustRatingRefinement = Interval.Closed[-1.0, 1.0]
```

Comment:
I see it duplicated

Reply:
`TrustRatingRefined` and `TrustRatingRefinement` have been replaced by `InternalTrustUpdateRefined` and `InternalTrustUpdateRefinement` respectively.